src/NumericalAlgorithms/FiniteDifference/PositivityPreservingAdaptiveOrder.hpp

```diff
@@ -29,11 +29,131 @@ class Index;
+                         56.63693768145594 * u[0]) +
+            u[2 * stride] * (10.664627625179254 * u[2 * stride] -
+                             9.781510753231265 * u[3 * stride] +
+                             3.783820939683476 * u[4 * stride] -
+                             25.682667845756164 * u[-stride] +
+                             13.59830711617153 * u[-2 * stride] -
+                             5.064486634342602 * u[-3 * stride] +
+                             1.5850428636128617 * u[-4 * stride] +
+                             33.99576779042882 * u[0]) +
+            u[3 * stride] * (2.5801312593878514 * u[3 * stride] -
+                             1.812843724346584 * u[4 * stride] +
+                             10.394184200706238 * u[-stride] -
+                             5.064486634342602 * u[-2 * stride] +
+                             1.6716163773782988 * u[-3 * stride] -
+                             0.4380794296257583 * u[-4 * stride] -
+                             14.626643302060115 * u[0]) +
+            u[4 * stride] * (0.5249097623867759 * u[4 * stride] -
+                             3.5773996341558414 * u[-stride] +
+                             1.5850428636128617 * u[-2 * stride] -
+                             0.4380794296257583 * u[-3 * stride] +
+                             0.07624062080823268 * u[-4 * stride] +
+                             5.336843456576288 * u[0]) +
+            u[-stride] * (25.393963433621668 * u[-stride] -
+                          31.738453392103736 * u[-2 * stride] +
+                          14.315575523531798 * u[-3 * stride] -
+                          5.422933317103013 * u[-4 * stride] -
+                          56.63693768145594 * u[0]) +
+            u[-2 * stride] * (10.664627625179254 * u[-2 * stride] -
+                              9.781510753231265 * u[-3 * stride] +
+                              3.783820939683476 * u[-4 * stride] +
+                              33.99576779042882 * u[0]) +
+            u[-3 * stride] * (2.5801312593878514 * u[-3 * stride] -
+                              1.812843724346584 * u[-4 * stride] -
+                              14.626643302060115 * u[0]) +
+            u[-4 * stride] * (0.5249097623867759 * u[-4 * stride] +
+                              5.336843456576288 * u[0]) +
+            33.758463458609164 * square(u[0]);
+        if (eight_to_the_alpha_9 * order_9_norm_of_top_modal_coefficient <=
+            order_9_norm_of_polynomial) {
+          return order_9_result;
+        }
+      }
+    }
+
+    if constexpr (Use7thOrder) {
+      const std::array order_7_result =
+          UnlimitedReconstructor<6>::pointwise(u, stride);
+
+      if (not PositivityPreserving or
+          LIKELY(order_7_result[0] > 0.0 and order_7_result[1] > 0.0)) {
+        const double order_7_norm_of_top_modal_coefficient =
+            0.15384615384615385 * square(0.3522952741702742 * u[stride] -
```

Comment:
I expected this to match Eq B.6 from your paper; it doesn't. Won't review further until you tell me who is wrong 😛

Reply:
The coeffs are in the correct ratio but have been multiplied by approximately  0.1328103086299076 🤷

Reply:
I have no idea where the numbers in the code came from... I do understand the numbers in the paper, so going with those :P